### PR TITLE
Use current time when overwriting model configuration.

### PR DESCRIPTION
**Pull request by whoisj**
This validates the change This validates the change made to `../core` wrt how model configuration mtime is handled.

[MR to ../core](https://github.com/triton-inference-server/core/pull/304)

### CODE REVIEW

File: qa/L0_lifecycle/lifecycle_test.py [hunk 1]
```diff
@@ -3254,6 +3255,83 @@ def test_concurrent_model_instance_load_sanity(self):
                     )
                     self.assertTrue(triton_client.is_server_ready())
 
+    def test_ensemble_config_overwite(self):
```
Comment (GuanLuo) [inline qa/L0_lifecycle/lifecycle_test.py:3258]: I don't see the test case is doing anything with ensemble, should probably change the name of the test case.

**Review by GuanLuo — COMMENTED** [review]
You may also want to address the Python style issue raised by CodeQL

File: qa/L0_lifecycle/lifecycle_test.py (hunk 1 above)
Comment (whoisj) [reply qa/L0_lifecycle/lifecycle_test.py]: :+1: good call, updated.

File: qa/L0_lifecycle/test.sh [hunk 2]
```diff
@@ -1947,3 +1947,38 @@ else
...
+SERVER_LOG="./inference_server_$LOG_IDX.log"
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+set +e
+python $LC_TEST LifeCycleTest.test_ensemble_config_overwite >>$CLIENT_LOG 2>&1
```
Comment (GuanLuo) [inline qa/L0_lifecycle/test.sh:1973]: reflect the new test case name

File: qa/L0_lifecycle/test.sh [hunk 3]
```diff
@@ -1947,3 +1947,38 @@ else
 fi
 
 exit $RET
+
+# LifeCycleTest.test_ensemble_config_overwite
```
Comment (GuanLuo) [inline qa/L0_lifecycle/test.sh:1951]: reflect the new test case name

File: qa/L0_lifecycle/lifecycle_test.py [hunk 4]
```diff
@@ -3254,6 +3255,83 @@ def test_concurrent_model_instance_load_sanity(self):
...
+        # Reload the model
+        triton_client.load_model(model_name)
+
+        # Ensure the model has been loaded w/ the expected (override) config.
+        updated_config = triton_client.get_model_config(model_name)
+        self.assertEqual(override_config, updated_config)
+
+        # Touch the local config.pbtxt and reload the file to ensure the local config
+        # is preferred because it has a more recent mtime.
+        Path(model_name + "/config.pbtxt").touch()
```
[the remark this answers is not in the document]
Comment (whoisj) [reply qa/L0_lifecycle/lifecycle_test.py]: good call, updated

**Review by GuanLuo — DISMISSED** [review]

**Comment by Tabrizian** [comment]
@whoisj you might need to resolve conflicts on this branch.

**Review by GuanLuo — APPROVED** [review]

**Pull request merged**